src/nbsap/views/indicators.py

```diff
@@ -5,6 +5,33 @@
 from auth import auth_required
 from nbsap import models
 from nbsap.forms import EuIndicatorForm, EuIndicatorMapForm
+from nbsap.models import sort_by_code, sort_by_type
+
+
+def get_most_relevant(indicator):
+    aichi_targets = []
+    if indicator.eu_indicator_aichi_strategy.exists():
+        for strategy in indicator.eu_indicator_aichi_strategy.all():
+            for aichi_target in strategy.aichi_targets.all():
+                if aichi_target not in aichi_targets:
+                    aichi_targets.append(aichi_target)
+    return aichi_targets
+
+
+def get_adjenct_indicators(current_indicator, indicators):
+    indicators = sort_by_type(sort_by_code(indicators))
+    previous_index = 0
+    next_index = 0
+    for index, indicator in enumerate(indicators):
+        if indicator == current_indicator:
+            previous_index = index - 1
+            next_index = index + 1
+    if previous_index < 0:
+        previous_index = len(indicators) - 1
+    if next_index > len(indicators) - 1:
+        next_index = 0
+
+    return indicators[previous_index], indicators[next_index]
```

Comment:
This can be done without iterating over `indicators`.

    current_index = indicators.index(current_indicator)
    previous_indicator = indicators[current_index - 1]  # this also works for current_index == 0
    try:
        next_indicator = indicators[current_index + 1]
    except IndexError:
        next_indicator = indicators[0]
    return previous_indicator, next_indicator